### ISSUE TITLE
Bug: some users expect the top-left dropdown menu to be related with the search box

### ISSUE THREAD

**Issue by sandriu** [TeamDignitas/dignitas#312]
![Dignitas-top-left-dropdown-list-expected-to-be-related-to-search](https://user-images.githubusercontent.com/1265882/150633225-e88d168e-ccc0-4c5f-8061-c57216c2827f.png)

That menu should become visually unrelated to the search box.

**Comment by CatalinFrancu** [comment]
Done and pushed to staging, please take a look.

I don't particularly like the left drawer now (on mobile). It expands on the entire height of the screen, but only uses a search box. Perhaps it would be better to only use the height of the search box?

Related TODO: The navbar searchbox could be wider, at all breakpoints.

**Comment by sandriu** [comment]
> Perhaps it would be better to only use the height of the search box?

Yes, it would be better. After all, most websites use such a behaviour (see the search box from medium.com for example).


> Related TODO: The navbar searchbox could be wider, at all breakpoints.

Agree too.

**Comment by sandriu** [comment]
I noticed the following problems on mobile:

- [ ] on light mode, the search box is surrounded by a white area (it should be the same colour as website header):
![Screenshot_20220209-125141](https://user-images.githubusercontent.com/1265882/153183994-88e19fa5-1b42-4c6c-8d14-31ff20b0f46e.jpg)

- [ ] on light mode, the top right menu has lost its dark background:
![Screenshot_20220209-125210](https://user-images.githubusercontent.com/1265882/153184851-a1d6894c-9c4d-412b-863b-72da3e30650c.png)

- [ ] on mobile, once the search box is expanded, it could occupy the full width of the page (it would help users see more text from the query they are typing in):
![Screenshot_20220209-124706](https://user-images.githubusercontent.com/1265882/153185414-e4c59549-25cc-4acf-989d-788a28f98181.jpg)

**Comment by CatalinFrancu** [comment]
Fixed the three items above and pushed to staging. Please note that the left and right menu now have different background colors.

**Comment by sandriu** [comment]
There's one more thing to fix now: when the search box is expanded, we have 2 search icons in the header, which is confusing:
![Screenshot_20220210-130613](https://user-images.githubusercontent.com/1265882/153403508-a095d135-7813-47f1-84da-75757bf1742b.png)


Most websites use the following interaction: the search button (or the menu button) transforms into a 'close' button (an X) once the search box (or the menu button) is clicked, and this close button is used to collapse the search box. See a live example on [TheGuardian.com](https://www.theguardian.com/international) (you must see the mobile version).
